ISMIR2021.tex

```diff
@@ -653,28 +653,28 @@ \section{Conference Program}
                     \begin{itemize}
                         \item Time: 1.30-3.00 AM on November 12, 2021 (UTC)
                         \item   Moderator: Gus Xia (NYU, Shanghai)
-                        \item   Panelists: Ajay Kapur (California Institute of the Arts, USA), Rujing Huang (University of Hong Kong, China), Lamtharn �Hanoi� Hantrakul (ByteDance/TikTok, China), Oriol �Uri� Nieto (Adobe, USA), Akira Maezawa (Yamaha, Japan)
+                        \item   Panelists: Ajay Kapur (California Institute of the Arts, USA), Rujing Huang (University of Hong Kong, China), Lamtharn “Hanoi” Hantrakul (ByteDance/TikTok, China), Oriol “Uri” Nieto (Adobe, USA), Akira Maezawa (Yamaha, Japan)
                     \end{itemize}
                 \item Panel 2: MIR Technologies and Education
                     \begin{itemize}
                         \item Time: 1.30-2.30 PM on November 12, 2021 (UTC)
                         \item  Moderator: Xavier Serra (Universitat Pompeu Fabra, Spain)
-                        \item  Panelists: Cynthia Liem (Delft University of Technology), Anja Volk (Utrecht University, Netherlands), Fran�ois Pachet (Spotify, France), Fabien Gouyon (Pandora, USA), Thomas Pr�tzlich (Skoove, Germany) 
+                        \item  Panelists: Cynthia Liem (Delft University of Technology), Anja Volk (Utrecht University, Netherlands), François Pachet (Spotify, France), Fabien Gouyon (Pandora, USA), Thomas Prätzlich (Skoove, Germany) 
                     \end{itemize}
             \end{itemize}
 
         \subsubsection{Masterclasses}
-            Masterclasses are sessions where panelists discuss topics related to getting into and working in the industry, and they are intended to familiarize current students and recent graduates with interviews and processes they may encounter when looking for MIR-style roles in industry. These sessions were not tied to sponsorship, i.e., the panelists were approached independent of their companies� participation in the conference.
+            Masterclasses are sessions where panelists discuss topics related to getting into and working in the industry, and they are intended to familiarize current students and recent graduates with interviews and processes they may encounter when looking for MIR-style roles in industry. These sessions were not tied to sponsorship, i.e., the panelists were approached independent of their companies’ participation in the conference.
              \begin{itemize}
                 \item Masterclass 1: CV
                     \begin{itemize}
                         \item   Time: 3.00-4.00 AM on November 12, 2021 (UTC)
-                        \item   Panelists: Christopher Bakes (ByteDance/TikTok,USA), Lamtharn �Hanoi� Hantrakul (ByteDance/TikTok, China)
+                        \item   Panelists: Christopher Bakes (ByteDance/TikTok,USA), Lamtharn “Hanoi” Hantrakul (ByteDance/TikTok, China)
```

Comment:
```suggestions
                        \item   Panelists: Christopher Bakes (ByteDance/TikTok, USA), Lamtharn “Hanoi” Hantrakul (ByteDance/TikTok, China)
```

Reply:
☝️ This should have create a commitable suggestion. Not sure why it didn't work. I'll check it later.

Reply:
Would probably be better to use `` '' for all quotation marks